hack/test-templates.sh

```diff
@@ -192,10 +192,11 @@ if [ "$got" != "$expected" ]; then
 fi
 
 INFO "Testing limactl copy command"
-tmpfile="$HOME/lima-hostname"
+tmpdir="$(mktemp -d ${TMPDIR:-/tmp}/lima-test-templates.XXXXXX)"
```

Comment:
The linter suggests to quote TMPDIR:
https://github.com/lima-vm/lima/actions/runs/13058754608/job/36436285027?pr=3163

Reply:
This linter issue can be detected quickly locally by running `make lint`. This will save the cost of running all the vms in the CI and save time for reviewers looking at the CI failures.

Reply:
The linter didn't catch that locally.